### PR TITLE
Feature/dtt 22 products and orders task 2

### PR DESCRIPTION
**Pull request by sebastianjohn11**
### orders list -create, delete, edit

### CODE REVIEW

File: src/components/OrdersForm.vue [hunk 1]
```diff
@@ -0,0 +1,121 @@
...
+    }
+  },
+  watch: {
+    items: function (newValue) {
+      if (Object.keys(newValue).length !== 0) {
+        this.order = newValue.order
+        this.description = newValue.description
+        this.price = newValue.price
+        this.edit = true
+        this.index = newValue.index
```
Comment (DijoMathew) [inline src/components/OrdersForm.vue:84]: You are using a this.index here and I am not seeing any index declared in data property

File: src/components/OrdersForm.vue [hunk 2]
```diff
@@ -0,0 +1,121 @@
...
+    showForm: {
+      get: function () {
+        return this.value
+      },
+      set: function (value) {
+        this.$emit('input', value)
+      }
+    },
+    items () {
+      console.log('working 1')
```
Comment (DijoMathew) [inline src/components/OrdersForm.vue]: Try to remove the things you have done for testing before pushing. I have already said it on previous Pr too

File: src/components/OrdersForm.vue [hunk 3]
```diff
@@ -0,0 +1,121 @@
...
+        this.index = newValue.index
+        this.showForm = true
+      }
+    }
+  },
+
+  methods: {
+    submitOrderForm () {
+      if (this.edit === false) {
+        this.$store.dispatch('orderFormSubmit', {
```
Comment (DijoMathew) [inline src/components/OrdersForm.vue]: For your previous ticket, you have used name spaces for sending data to a particular vuex module, then why you didn't used it here? Try to make use of it when you got a vuex module.

File: src/components/OrdersForm.vue [hunk 4]
```diff
@@ -0,0 +1,121 @@
...
+          description: this.description,
+          price: this.price,
+          index: this.index
+        })
+        this.close()
+        console.log('working 2')
+      }
+    },
+    close () {
+      this.$store.dispatch('clearItems',
```
Comment (DijoMathew) [inline src/components/OrdersForm.vue]: Here as well.

**Review by DijoMathew — COMMENTED** [review]
When you are creating a Pr just mention your ticket number and under it give a brief description of the things you have done.

File: src/components/OrdersForm.vue (hunk 1 above)
Comment (sebastianjohn11) [reply src/components/OrdersForm.vue:84]: this.index is not required here ,i removed it

File: src/components/OrdersForm.vue [hunk 5]
```diff
@@ -90,32 +88,31 @@ export default {
...
           price: this.price
         })
         this.close()
       } else {
-        this.$store.dispatch('submitEditForm', {
+        this.$store.dispatch('orders/submitEditForm', {
           order: this.order,
           description: this.description,
           price: this.price,
           index: this.index
```
Comment (DijoMathew) [inline src/components/OrdersForm.vue:104]: What is this index then ? Try to compare it with what you have done previously and do it accordingly, you require that index for editing right?

**Review by DijoMathew — APPROVED** [review]

**Pull request merged**